core/src/main/java/org/apache/carbondata/core/datastore/row/CarbonRow.java

```diff
@@ -49,6 +49,14 @@ public CarbonRow(Object[] data, Object[] rawData) {
     return data;
   }
 
+  public Object[] getKey(int[] sortColumnIndex) {
+    Object[] key = new Object[sortColumnIndex.length];
+    for(int i = 0; i < sortColumnIndex.length; i++){
```

Comment:
add ` ` after `for`

Reply:
But the sort column fields are diverse in the carbonrow. I can't copy a continuous intervels objects in the array.

Reply:
Please check sortColumnIndex. I think it is continuous. 
we group all the sort columns at the beginning of schema. so data also should be arranged as per schema.